src/apis/evm/api.ts

```diff
@@ -638,12 +638,13 @@ export class EVMAPI extends JRPCAPI {
    * @param amount The amount being exported as a {@link https://github.com/indutny/bn.js/|BN}
    * @param assetID The asset id which is being sent
    * @param destinationChain The chainid for where the assets will be sent.
-   * @param toAddresses The addresses to send the funds
-   * @param fromAddresses The addresses being used to send the funds from the UTXOs provided
-   * @param changeAddresses The addresses that can spend the change remaining from the spent UTXOs
-   * @param asOf Optional. The timestamp to verify the transaction against as a {@link https://github.com/indutny/bn.js/|BN}
+   * @param fromAddressesHex The addresses to send the funds (hex)
+   * @param fromAddressesBech The addresses being used to send the funds from the UTXOs provided
+   * @param toAddresses An array of addresses as {@link https://github.com/feross/buffer|Buffer} who recieves the AVAX
```

Comment:
`AVAX`, I think it's `CAM`

Reply:
Ideal solution would be "Native Asset" all over the places but we want to be less invasive.
Beside this the comment was wrong and simply copied from the implementation.
Pls. always keep in mind that caminojs is capable for different native assets, also for AVAX.